test/DynamicDependency/Test_WinRT/pch.h

```diff
@@ -32,10 +32,8 @@
 #include <MddBootstrap.h>
 #include <MddBootstrapTest.h>
 
-#include "CppUnitTest.h"
-#include "CppUnitTestX.h"
+#include <WexTestClass.h>
```

Comment:
nit: Why is the above MsixDynamicDepency in the middle of the winrt headers instead of grouped outside it? ;P